src/hotspot/share/gc/parallel/psOldGen.cpp

```diff
@@ -294,6 +294,7 @@ void PSOldGen::resize(size_t desired_free_space) {
   new_size = clamp(new_size, min_gen_size(), max_gen_size());
 
   new_size = align_up(new_size, alignment);
+  new_size = clamp(new_size, min_gen_size(), max_gen_size());
```

Comment:
Is that a merge conflict and are min/max_gen_size() alligned?

Reply:
Oops, that's a leftover. Thank you for pointing out.